sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawContainer.java

```diff
@@ -15,7 +16,12 @@ private RawContainer(
             @JsonProperty("image-name") String imageName,
             @JsonProperty("rlimits") WriteOnceLinkedHashMap<String, RawRLimit> rlimits) {
         this.imageName = imageName;
-        this.rlimits = rlimits;
+
+        if (rlimits != null) {
```

Comment:
There may be a way of more gracefully handling this through the YAML mapper, but this was quicker.